### PR TITLE
Update to Eclipse 2023-12

### PR DESCRIPTION
**Pull request by TruongQuangSB**

### CODE REVIEW

File: java/bundles/org.eclipse.set.releng.target/org.eclipse.set.releng.target.target [hunk 1]
```diff
@@ -3,83 +3,85 @@
...
+			<unit id="org.apache.commons.jxpath" version="1.3.0.v200911051830"/>
+			<unit id="com.ibm.icu" version="74.1.0"/>
+			<unit id="com.sun.jna" version="5.13.0.v20230812-1000"/>
+			<unit id="com.sun.jna.platform" version="5.13.0"/>
+			<unit id="bcpg" version="1.77.0"/>
+			<unit id="bcprov" version="1.77.0"/>
+			<repository location="https://download.eclipse.org/releases/2023-12/"/>
+		</location>
+		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="slicer" includeSource="true" type="InstallableUnit">
+			<unit id="javax.inject" version="0.0.0"/>
```
Comment (TruongQuangSB) [inline java/bundles/org.eclipse.set.releng.target/org.eclipse.set.releng.target.target:47]: javax.inject still need for emf ecp

File: java/bundles/org.eclipse.set.releng.target/org.eclipse.set.releng.target.target [hunk 2]
```diff
@@ -3,83 +3,106 @@
...
+			<repository location="https://download.eclipse.org/eclipse/updates/4.30/"/>
 		</location>
-
 		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="slicer" includeSource="true" type="InstallableUnit">
 			<unit id="org.eclipse.babel.nls_eclipse_de.feature.group" version="0.0.0"/>
 			<unit id="org.eclipse.babel.nls_modeling.emf_de.feature.group" version="0.0.0"/>
 			<repository location="https://download.eclipse.org/technology/babel/update-site/R0.20.0/2022-12/"/>
 		</location>
-
 		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="slicer" includeSource="true" type="InstallableUnit">
```
Comment (pstuecker) [inline java/bundles/org.eclipse.set.releng.target/org.eclipse.set.releng.target.target:96]: Xtend 2.34 is released and should be used instead

**Review by pstuecker — CHANGES_REQUESTED** [review]

File: java/bundles/org.eclipse.set.releng.target/org.eclipse.set.releng.target.target (hunk 2 above)
Comment (TruongQuangSB) [reply java/bundles/org.eclipse.set.releng.target/org.eclipse.set.releng.target.target:95]: ok

**Pull request merged**